### ISSUE TITLE
bug[accessibility]: No labels for navigation items

### ISSUE THREAD

**Issue by moan0s** [Razeeman/Android-SimpleTimeTracker#404]
## Steps to reproduce

- Activate Android Talkback and open the app.
- Tab through the main navigation

## What happens

Screen reader says "Tab 1", "Tab 2", Tab 3", Tab 4" and then continues with  "My activity name" (this)

## What I would expect

Screen reader reads "Timer", "History", "Statistics", "Settings" and then continues with "My activity name"

Feel free to ping me if you need help to reproduce it

**Comment by Razeeman** [comment]
Hello! Thank you very much for pointing this out. Pushed a fix, will be in the next release.
